src/Polysemy/Internal.hs

```diff
@@ -72,6 +72,60 @@ import Polysemy.Internal.Union
 -- is the order in which you call the interpreters that determines the
 -- monomorphic representation of the @r@ parameter.
 --
+-- While order of effects in effect stack by itself is not important, order of
```

Comment:
Doesn't the order of interpreters *determine* the order of the effect stack? (what I mean is that the effect stack is important - it would be more accurate to say that you determine its order via the order of interpreters) Sorry if I'm just misunderstanding!

Reply:
Yeah, @googleson78 has the right idea here!

Reply:
Yes - I was thinking about mappings, ignoring and stuff that changes stack in a little bit different ways and didn't use the right words :sweat_smile: